make-utils.jl

```diff
@@ -1,6 +1,9 @@
-using Base64, Pkg
-Pkg.add(url="https://github.com/TuringLang/TuringTutorials", rev="artifacts")
-using TuringTutorials
+using Base64
+using LibGit2
+using Pkg
+
+const OUTPUT_DIR = "TuringTutorialsOutput"
+clone("https://github.com/TuringLang/TuringTutorialsOutput", OUTPUT_DIR)
```

Comment:
Coming back to the discussion in https://github.com/TuringLang/turing.ml/pull/26/files#r846601425 about removing `OUTPUT_DIR`: It seems `OUTPUT_DIR` is only used in this specific function below but not anywhere else. So it seems we could just remove the global constant and move `clone(...)` to that function? That would also solve any problems with rerunning the function (since currently the cloned repo is removed as discussed in the other thread).

Reply:
@rikhuijzer If you rather want to just merge the PR, I can open a follow-up PR that removes the global constant.